cleanlab_studio/internal/util.py

```diff
@@ -63,3 +64,131 @@ def check_none(x: Any) -> bool:
 
 def check_not_none(x: Any) -> bool:
     return not check_none(x)
+
+
+def _get_autofix_default_params() -> dict:  # Studio team port to backend
```

Comment:
allow string options (rethink names):

"optimized_training_data"

"all_issues"

Reply:
suggested_actions strategy should relabel all label issues, drop outliers, and extra copies of near duplicates. Nothing done to ambiguous examples or other issues